css-gaps-1/Overview.bs

```diff
@@ -0,0 +1,601 @@
+</pre>
+
+Introduction {#intro}
+=====================
+
+	<em>This section is not normative.</em>
+
+	[[CSS-MULTICOL-1#column-gaps-and-rules]] allows for rules to be drawn
+	between columns in a <a>multicol container</a>. This specification expands
+	upon the 'column-rule-width', 'column-rule-style', and 'column-rule-color'
+	properties, adding equivalents in the row direction, expanding their
+	application to other container layouts, and giving advanced control over
+	where and how gap decorations are painted.
+
+Value Definitions {#values}
+---------------------------
+
+	This specification follows the <a href="https://www.w3.org/TR/CSS2/about.html#property-defs">CSS property definition conventions</a> from [[!CSS2]]
+	using the <a href="https://www.w3.org/TR/css-values-3/#value-defs">value definition syntax</a> from [[!CSS-VALUES-3]].
+	Value types not defined in this specification are defined in CSS Values &amp; Units [[!CSS-VALUES-3]].
+	Combination with other CSS modules may expand the definitions of these value types.
+
+	In addition to the property-specific values listed in their definitions,
+	all properties defined in this specification
+	also accept the <a>CSS-wide keywords</a> as their property value.
+	For readability they have not been repeated explicitly.
+
+Gap decorations {#gap-decorations}
+==================================
+
+	Various layouts in CSS such as
+	<a>multicol containers</a>, <a>flex containers</a>, <a>grid containers</a>, and <a>masonry containers</a>
+	position child boxes adjacent to each other with gaps, also known as <a>gutters</a>, between them.
+	Each such gap may contain a <dfn>gap decoration</dfn> which authors
+	may use to provide a visual separator between adjacent boxes.
+
+	<div class="example">
+		<pre>
+			.grid-with-spans {
+			  display: grid;
+			  grid-template: repeat(4, 100px) / repeat(4, 100px);
+			  gap: 20px;
+			  row-rule: 6px solid red;
+			  column-rule: 6px solid blue;
+			}
+		</pre>
+		<img src="images/example-grid-with-spans.png">
+		<figcaption>A grid with spanning items and gap decorations.
+	</div>
+
+	<div class="example">
+		<pre>
+			.flex {
+			  display: flex;
+			  flex-wrap: wrap;
+			  gap: 20px;
+			  width: 500px;
+			  row-rule: 6px solid red;
+			  column-rule: 6px solid blue;
+			}
+		</pre>
+		<img src="images/example-flex.png">
+		<figcaption>A flexbox with gap decorations.
+	</div>
+
+Layout and painting {#layout-painting}
+--------------------------------------
+
+	This specification does not generally define the geometry of gaps in container layouts;
+	that is left to other specifications.
+	See [[CSS-ALIGN-3#gaps]] for definitions pertaining to
+	<a>multicol containers</a>, <a>flex containers</a>, and <a>grid containers</a>.
+
+	<a>Gap decorations</a> do not take up space.
+	That is, the presence or thickness of a <a>gap decoration</a> will not alter the placement of anything else.
```

Comment:
Is "thickness" the best word here? "dimensions" seems more generic. Or should you use "width" to better align with the properties defined below?

Reply:
For context, this paragraph was copied verbatim from https://drafts.csswg.org/css-multicol-1/#column-gaps-and-rules. But I agree with this and the next couple of suggested changes to the wording, so I'll go ahead and make them.